tests/db_list.py

```diff
@@ -0,0 +1,29 @@
+"""List of running databases"""
```

Comment:
In that case, every database would need a distinct approach, so I guess we could solve this problem in a separate PR. Since these checks have been working well so far, I think we could do with them for now.

Reply:
> Since these checks have been working well so far

Actually, no. I've had a problem with these checks (`ping_localhost` passed, however some dbs were not available). Unfortunately, it was some time ago and I already fixed it (probably by restarting db containers) so I don't know how to reproduce it.

But I agree. `ping_localhost` was in the codebase before this PR and our approach to testing db availability should be changed in another PR.